docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -330,6 +330,17 @@ and in the case of using the Maven configuration instead of `application.propert
 ----
 ====
 
+=== Working with Random and SplittableRandom
+
+If your native application is initialized at build time and has static `java.util.Random` or `java.util.SplittableRandom` instances then its native build will fail because embedding such instances in native image will result in the same cached seed value be reused which will negatively affect the entropy (randomness) quality of the produced values.
```

Comment:
I think this is highly confusing: `If your native application is initialized at build time`.

You don't have the option in Quarkus to initialize "your application" at build time or runtime.

Reply:
@gsmet Sure, this is a general point I've tried to convey from the tutorial.

So. may be just `If your native application has static ...`, omitting `is initialized at build time`, please suggest an update.